For old glibc's one might have to disable AIO, however KVM always
requires compatfd to be built.

  - Rene Rebe <rene@exactcode.de>

--- kvm-83/qemu/Makefile.vanilla	2009-02-04 20:37:40.000000000 +0000
+++ kvm-83/qemu/Makefile	2009-02-04 20:38:03.000000000 +0000
@@ -62,6 +62,10 @@
 
 ifdef CONFIG_AIO
 BLOCK_OBJS += compatfd.o
+else
+ifeq ($(USE_KVM), 1)
+BLOCK_OBJS += compatfd.o
+endif
 endif
 
 ######################################################################
--- kvm-83/qemu/Makefile.target.vanilla	2009-02-04 20:39:02.000000000 +0000
+++ kvm-83/qemu/Makefile.target	2009-02-04 20:39:16.000000000 +0000
@@ -622,6 +622,10 @@
 
 ifdef CONFIG_AIO
 OBJS+=compatfd.o
+else
+ifeq ($(USE_KVM), 1)
+OBJS+=compatfd.o
+endif
 endif
 
 LIBS+=-lz
